roles/install-devstack/tasks/main.yml

```diff
@@ -79,13 +79,15 @@
 
       # Delete all rules in chain openstack-INPUT to allow unlimited access from nested vm
       iptables -F openstack-INPUT || true
-
-      # Remove distutils packages that conflict with pip packages that devstack needs to bigip_software_install
-      apt remove python3-httplib2
     executable: /bin/bash
     chdir: '{{ ansible_user_dir }}/workspace'
   environment: '{{ zuul | zuul_legacy_vars }}'
 
+- name: Remove distutils packages that conflict with pip packages that devstack needs to bigip_software_install
```

Comment:
This looks like a better way to do it.  But it looks like I had a typo in my original comment (some stray 'bigip_software' string -- This should be:
  - name: Remove distutils packages that conflict with pip packages that devstack needs to install

Dunno where that 'bigip_software_install' came from (though I guess I did it).

Reply:
Thanks! Changed the comment.